src/plugins/auto_batch/src/compiled_model.cpp

```diff
@@ -39,63 +38,44 @@ CompiledModel::~CompiledModel() {
     m_worker_requests.clear();
 }
 
-unsigned int CompiledModel::ParseTimeoutValue(const std::string& s) {
-    auto val = std::stoi(s);
-    if (val < 0)
-        IE_THROW(ParameterMismatch) << "Value for the " << CONFIG_KEY(AUTO_BATCH_TIMEOUT) << " should be unsigned int";
-    return val;
+std::shared_ptr<ov::IRemoteContext> CompiledModel::get_context() const {
+    return m_compiled_model_without_batch->get_context();
 }
 
-std::shared_ptr<InferenceEngine::RemoteContext> CompiledModel::GetContext() const {
-    return m_model_without_batch->GetContext();
-}
-
-InferenceEngine::IInferRequestInternal::Ptr CompiledModel::CreateInferRequestImpl(
-    InferenceEngine::InputsDataMap networkInputs,
-    InferenceEngine::OutputsDataMap networkOutputs) {
-    auto workerRequestPtrAndId = GetWorkerInferRequest();
-    return std::make_shared<SyncInferRequest>(networkInputs,
-                                              networkOutputs,
-                                              workerRequestPtrAndId.first,
-                                              workerRequestPtrAndId.second,
-                                              m_device_info.batch_for_device,
-                                              m_batched_inputs,
-                                              m_batched_outputs);
-}
-
-InferenceEngine::IInferRequestInternal::Ptr CompiledModel::CreateInferRequestImpl(
-    const std::vector<std::shared_ptr<const ov::Node>>& inputs,
-    const std::vector<std::shared_ptr<const ov::Node>>& outputs) {
-    if (!this->_plugin || !_plugin->IsNewAPI())
+std::shared_ptr<ov::ISyncInferRequest> CompiledModel::create_sync_infer_request() const {
+    if (!get_plugin())
```

Comment:
I suppose we can remove this check? We don't need a plugin to create an inference request.
The check was because of old API